packages/localstorage/localstorage.js

```diff
@@ -21,7 +21,33 @@ try {
 } catch (ignored) {}
 
 if (key === retrieved) {
-  Meteor._localStorage = storage;
+  if (Meteor.isServer) {
+    Meteor._localStorage = storage;
+  } else {
+    // IE11 doesn't handle properly attempts to change methods of the
```

Comment:
A new commit is fine - we'll squash when merging (and it's nice to see the commit history while working/reviewing through the PR). A commit message like "Adjusted proxy method explanation comment" or something similar would work. Thanks!